### ISSUE TITLE
Inferior results (FastMRI)

### ISSUE THREAD

**Issue by wafaAlghallabi** [z-fabian/HUMUS-Net#5]
Hello,

Thank you very much for your effort.

I have a question regarding FastMRI results that you've reported in the paper. I trained your Model using the same settings but I got a lower SSIM score on the validation set, 0.8766 (compared to 0.8954).

May I know if you have any idea what could be the reason behind this inferior?

Thanks.

**Comment by z-fabian** [comment]
Hi wafaAlghallabi,

Can you try downloading our pre-trained checkpoints and see if you get the reported performance? Those models have been evaluated on the official fastMRI test dataset, so they should be working properly.
Let me know how it goes!

**Comment by wafaAlghallabi** [comment]
Thank you for your reply.

One question though, how can I evaluate the model on the test set?. I tried setting the "test_path" in `eval_humus_fastmri.py` and it didn't work. 

Thank you.

**Comment by wafaAlghallabi** [comment]
> Hi wafaAlghallabi,
> 
> Can you try downloading our pre-trained checkpoints and see if you get the reported performance? Those models have been evaluated on the official fastMRI test dataset, so they should be working properly. Let me know how it goes!

Thank you for your reply.

One question though, how can I evaluate the model on the test set?. I tried setting the "test_path" in eval_humus_fastmri.py and it didn't work.

Regards.
